draft-ietf-tls-sni-encryption.md

```diff
@@ -91,30 +90,32 @@ server context. The current SNI extension specification can be
 found in [@?RFC6066].
 
 The SNI specification allowed for different types of server names,
-but only the "hostname" variant was standardized and deployed. In that
+though only the "hostname" variant was standardized and deployed. In that
 variant, the SNI extension carries the domain name of the target
-server. The SNI extension is carried in clear text in the TLS "Client Hello"
-message. 
+server. The SNI extension is carried in clear text in the TLS "ClientHello"
+message.
 
 ## Unanticipated usage of SNI information {#snileak}
 
-The SNI was defined to facilitate management of servers, but the
+The SNI was defined to facilitate management of servers, though the
 developer of middleboxes soon found out that they could take
```

Comment:
Wording still seems awkward,  maybe it should be 'though the developers of middleboxes'

Reply:
Yep, good catch!